buildspec-vw.yml

```diff
@@ -0,0 +1,97 @@
+version: 0.2
+
+env:
+  variables:
+    CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
```

Comment:
This instance type is not used yet. Talked to Siddhartha offline. The container does not have sagemaker integ tests yet. Please make sure not using hardcoded instance type in release build since instance availability varies among regions.

Reply:
Thanks for pointing it out. I'll add a small comment here